src/ckeditor.js

```diff
@@ -9,7 +9,11 @@ export default {
 	name: 'ckeditor',
 
 	render( createElement ) {
-		return createElement( this.tagName );
+		return createElement(this.tagName, {
+			domProps: {
+				innerHTML: this.value || ''
```

Comment:
Does this work when `tagName` is `textarea`?

Reply:
I'm worried because setting the value of an input like this works only once (the second one will be ignored). In theory, we do it only once but still, this is not the right way IMO.

Reply:
Also... does this whole `innerHTML` thing work when a content with/without entities is passed as a value?

Reply:
I've built a sample with `tagName` set to `textarea`. The editor seems to work properly, no error occurs on init and on destroy.